sass/partials/_intro.scss

```diff
@@ -0,0 +1,56 @@
+.intro-content {
+	margin-top: 120px;
+	padding: 0px 20px;
+	@include min-screen($screen_sm_min) {
+		margin-top: 200px;
+	}
+}
+
+.primary-tagline {
+	font-size: 2em;
+	color: white;
+	font-weight: $base_font_weight_semi_bold;
+	text-align: center;
+	@include min-screen($screen_sm_min) {
+		font-size: 3em;	
+	}
+}
+
+.secondary-tagline {
+	color: white;
+	opacity: 0.7;
+	margin-top: 20px;
+	font-size: 1em;
+	font-weight: $base_font_weight_light;
+	text-align: center;
+	line-height: 1.2em;
+	@include min-screen($screen_sm_min) {
+		font-size: 1.3em;
+		line-height: 1.3em;
+	}
+}
+
+.lp-button-wrapper {
+	padding: 0;
+	width: 100%;
+}
+
+.lp-button {
+	border: white solid 2px;
+	color: white;
+	background: black;
+	margin: 110px auto auto auto;
+	text-align: center;
+	display: block;
+	background: $dark_green;
+	padding: 20px 30px;
+	@include border-radius(10px);
+	font-size: 1.5em;
+	@include hover-animation(all);
+	cursor: pointer;
```

Comment:
the `@include`s are usually done at the top before other styling just for ease of reading.